.github/workflows/test.yaml

```
@@ -0,0 +1,28 @@
+      - name: JDK 17 설정
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+          cache: gradle
+
+      - name: 테스트 실행
+        run: ./gradlew test
+
+      - name: Codecov 테스트 커버리지 업로드
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)